internal/vendor/wiz.go

```diff
@@ -141,10 +143,28 @@ var Environment = []string{
 type ProjectCloudOrganizationLinkInput struct {
 	CloudOrganization string         `json:"cloudOrganization"`
 	Environment       string         `json:"environment"` // enum Environment
+	ResourceGroups    []string       `json:"resourceGroups"`
 	ResourceTags      []*ResourceTag `json:"resourceTags"`
 	Shared            bool           `json:"shared"`
 }
 
+// ProjectCloudAccountLinkInput struct
+type ProjectCloudAccountLinkInput struct {
+	ID             string         `json:"cloudAccount"`
+	Environment    string         `json:"environment"` // enum Environment
+	ResourceGroups []string       `json:"resourceGroups"`
+	ResourceTags   []*ResourceTag `json:"resourceTags"`
+	Shared         bool           `json:"shared"`
```

Comment:
Since this is an optional bool, the type should be a pointer to bool instead of bool.

```suggestion
	Shared         *bool          `json:"shared"`
```

This will prevent the default initialized value of bool, false, from being used and allow a null to be passed.